main.js

```diff
@@ -0,0 +1,6 @@
+let message ="hi this is my homework" 
+let x = ( message.length +100 - (message.length % 100) )
+let cost = 1
+console.log (cost * x/100 )
```

Comment:
Spacing and semicolon 
```suggestion
console.log(cost * x / 100);
```